Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -209,20 +209,15 @@ void JitArm64::FallBackToInterpreter(UGeckoInstruction inst)
   }
 }
 
-void JitArm64::HLEFunction(UGeckoInstruction inst)
+void JitArm64::HLEFunction(u32 hook_index)
 {
   gpr.Flush(FlushMode::FLUSH_ALL);
   fpr.Flush(FlushMode::FLUSH_ALL);
```

Comment:
Is there a chance that the first instruction is an integer instruction? If so, "FlushCarry();" might need to be called as well.

Reply:
FlushCarry added.